ddtrace/settings/integration.py

```diff
@@ -57,15 +57,17 @@ def _get_analytics_settings(self):
         # Set default analytics configuration, default is disabled
         # DEV: Default to `None` which means do not set this key
         # Inject environment variables for integration
-        _ = os.environ.get("DD_TRACE_%s_ANALYTICS_ENABLED" % self.integration_name.upper()) or get_env(
-            self.integration_name, "analytics_enabled"
+        _ = os.getenv(
+            "DD_TRACE_%s_ANALYTICS_ENABLED" % self.integration_name.upper(),
+            os.getenv("DD_%s_ANALYTICS_ENABLED" % self.integration_name.upper()),
         )
         analytics_enabled = asbool(_) if _ is not None else None
 
         analytics_sample_rate = float(
-            os.environ.get("DD_TRACE_%s_ANALYTICS_SAMPLE_RATE" % self.integration_name.upper())
-            or get_env(self.integration_name, "analytics_sample_rate")
-            or 1.0
+            os.getenv(
+                "DD_TRACE_%s_ANALYTICS_SAMPLE_RATE" % self.integration_name.upper(),
+                os.getenv("DD_%s_ANALYTICS_SAMPLE_RATE" % self.integration_name.upper(), default=1.0),
```

Comment:
in the places in this script where we have `os.getenv("", os.getenv()`, can do do:

`os.getenv("", default=os.getenv("", default=))`, it will make it easier to read than:

> os.getenv("DD_" + name.upper() + "_" + "SERVICE", os.getenv("DD_" + name.upper() + "_" + "SERVICE_NAME", default=None))